### DISCUSSION TITLE
Adjacent boxplots

### DISCUSSION THREAD

**Discussion by royfrancis** [observablehq/plot#2216] [Q&A]
The “fill” argument for box mark stacks boxplots above each other. Is it possible to have them adjacent to each other so don’t overlap?

This is how it looks like now:

![Screenshot 2024-10-31 at 13 35 57](https://github.com/user-attachments/assets/083afba8-696b-4ec7-bb55-5f51b0ee2700)

This is what I have in mind:

![Screenshot 2024-10-31 at 13 37 41](https://github.com/user-attachments/assets/e8270dbd-2d43-4d1c-927e-5e485161b4fc)

Here is some sample code:

```js
filteredDiamonds = diamonds.filter(d => 
  (d.color === 'D' || d.color === 'E' || d.color === 'F') &&
  (d.clarity === 'I1' || d.clarity === 'IF' || d.clarity === 'SI1')
)

Plot.plot({
  title: "Boxplot",
  marks: [
    Plot.boxY(filteredDiamonds, {
      x: "cut",
      y: "depth",
      fill: "clarity",
      fx: "color",
    }),
    Plot.axisX({tickRotate: -45})
  ],
  grid: true,
  color: {
    legend: true
  }
})
```

Here is an example working document: [Adjacent boxplots / workbook | Observable 2](https://observablehq.com/d/787f059ccb507e9f)

**Comment by Fil** [comment]
see #2217
